forms/tests/test_utils.py

```diff
@@ -40,8 +40,10 @@ def test_form_cloning(basic_template_form):
     new_fieldtype_count = FieldType.objects.all().count()
 
     assert new_form.id != basic_template_form
-    assert new_section_count == section_count * 2
-    assert new_field_count == field_count * 2
+    # Cloning basic template form should add 7 sections.
+    assert new_section_count == section_count + 7
+    # Cloning basic template form should add 23 fields.
+    assert new_field_count == field_count + 23
```

Comment:
Slight improvement could be defining this as `BASIC_TEMPLATE_FIELD_COUNT = 23` and then in the assert `field_count + BASIC_TEMPLATE_FIELD_COUNT`